frontend/src/common/types/permissions/permissions-group-creation-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+type PermissionsGroupCreationDto = {
+  checkbox: JSX.Element;
```

Comment:
I believe it's not the best way to pass component within data values - you can specify Checkbox as Cell presentation and just provide custom value for it (it would be `checked/unchecked` depending on whether there is a certain permission in current group or not)

Reply:
Not resolved. I tried doing it as you adviced, through headers & custom cell, but it didn't work out for me. Maybe I was trying in wrong way.

Reply:
You may take a look how similar thing was done [here](https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/83/files#diff-e16ced115b9a58a1ddfaa12abda1d014f1d92270e9170d0dd2fd3c576f1862eeR1) - in that PR there is example for custom component cell providing and for props processing for that cell